projects/powerflows/src/lib/powerflows.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
```

Comment:
Because of intellectual property please add for our every file (typescript, styles and if possible to html as well) a license header, like in https://github.com/powerflows/powerflows-dmn/blob/master/src/main/java/org/powerflows/dmn/domain/model/Decision.java

Reply:
By mistake I did it in PR solving #4